src/Pyz/Client/MultiCart/MultiCartConfig.php

```diff
@@ -40,6 +40,7 @@ public function getQuoteFieldsAllowedForCustomerQuoteCollectionInSession(): arra
             QuoteTransfer::NAME,
             QuoteTransfer::IS_DEFAULT,
             QuoteTransfer::CUSTOMER_REFERENCE,
+            QuoteTransfer::QUOTE_APPROVALS,
```

Comment:
is it already mentioned at IG?
otherwise we need top update it

Reply:
https://github.com/spryker/spryker-docs/pull/1947